setup.py

```diff
@@ -0,0 +1,44 @@
+from setuptools import setup, find_packages, find_namespace_packages
+import pathlib
+
+here = pathlib.Path(__file__).parent.resolve()
+
+# workaround a bug introduced by pyproject.toml
+# https://github.com/pypa/pip/issues/7953#issuecomment-645133255
+import site, sys; site.ENABLE_USER_SITE = True
+
+setup(
+  name='spinalcordtoolbox-data-<dataset>',
+  description='Part of https://github.com/neuropoly/spinalcordtoolbox',
+  long_description=(here / 'README.md').read_text(encoding='utf-8'),
+  long_description_content_type='text/markdown',
+  author='Neuropoly',
+  author_email='neuropoly@googlegroups.com',
+  url='https://spinalcordtoolbox.com/',
+  project_urls={
+      'Source': 'https://github.com/sct-data/<dataset>',
+      #'Documentation': '',
+  },
+  #license='CC-BY-NC', ??
+  #license_files=[ ... ] # TODO?
```

Comment:
We settled on CC in the meeting, right?

Reply:
I'm not totally sure.

@jcohenadad do you want CC BY-NC-SA 4.0 or CC BY-SA 4.0 or CC BY 4.0 ?

Also licensing is a thorny legal question. We can't just slap a license on a work; at a minimum we need to contact everyone in the git log and get their thoughts, otherwise the work is proprietary to everyone who has ever worked on it, by default.

I guess for the template we can fill in an aspirational one, but for the actual datasets it's a lot trickier.

Reply:
I'm OK with `CC BY 4.0`

Reply:
```suggestion
  #license='CC BY 4.0',
  #license_files=[ LICENSE.txt ]
```

Reply:
TODO: Fetch CC and put as file